src/main/java/org/candlepin/subscriptions/db/model/SubscriptionCapacity.java

```diff
@@ -40,6 +35,8 @@
 @AllArgsConstructor(access = AccessLevel.PRIVATE)
 @Getter
 @Setter
+@EqualsAndHashCode
+@ToString
```

Comment:
We can replace a lot of these annotations with the `@Data` annotation

https://projectlombok.org/features/Data